src/lib/ngsi/ContextAttribute.h

```diff
@@ -39,20 +39,42 @@
 
 /* ****************************************************************************
 *
+* ContextAttributeValueType - 
+*/
+// FIXME P10: we should use the same Enum (in a separate .h) file for Metadata, ContextAttribute, and CompoundValueNode
+typedef enum ContextAttributeValueType
```

Comment:
The 3 enums in ContextAttribute, Metadata and CompoundNodeValue should be unified in only one, living in its own .h/.cpp.